index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+
+    <header>
+        <div>
+            <nav id="head">
+                <p>Safar Ali</p>
+                <img src="../Portfolio/images/Union.png" alt="burger-icon">
+            </nav>
+            <div id="name">
+                <h2>I'm Safar Ali <br> A thriving <br> Software Developer </h2>
+            </div>
+            <div id="para">
+                <p>I am a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+            <div id="text">
+                <p>Let us Connect</p>
+            </div>
+            <div id="icons">
+                <ul>
+                    <li><img src="../Portfolio/images/github.png" alt=""></li>
+                    <li><img src="../Portfolio/images/Linkedin.png" alt=""></li>
+                    <li><img src="../Portfolio/images/twitter.png" alt=""></li>
+                   <li><img src="../Portfolio/images/Vector.png" alt=""></li>
+                </ul>
+            </div>
+        </div>
+    </header>
+
+</body>
```

Comment:
- [ ] Kindly, remove the headline in the `header` container. The header container should contain only the toolbar(navigation) of your pages. Kindly, fix it by referencing on this example below: 
```
<header>
     ....
</header>
<section class="...">
     <div class="...">
          <div id="name">
          </div>
          .....
     </div>
</section>

Reply:
@RolandM99 can you please explain a bit more regarding this part? I didn't understand what do you want me to change in this part.
looking forward!
Thanks in advance.

> * [ ]  Kindly, remove the headline in the `header` container. The header container should contain only the toolbar(navigation) of your pages. Kindly, fix it by referencing on this example below:
> 
> ```
> <header>
>      ....
> </header>
> <section class="...">
>      <div class="...">
>           <div id="name">
>           </div>
>           .....
>      </div>
> </section>
> ```

Reply:
Hi @safar1212 , Great to reach out to me for this feature. As you can see, in your code the `header` tag contains all the elements you had to implement (the navigation or toolbar and the headline). But depending on project requirements, you should put the headline outside the `header` tag. they are two different sections. the headline has like container a` section`  tag. for this part, you don't need to delete anything, you can just cut and copy your code from line 18 up line 35 in the section tag I implemented above. as you can see inside I started with the div tag you used in your code to make it easy and understandable

Reply:
@RolandM99 
Thank you for your great explanation. I will do the same and put it in review.